app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  after_save :post_increment
+
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
```

Comment:
✅ The associations between models are set up correctly according to the ERD diagram 👍🏻